### PR TITLE
Created new table Global stock admin in PS2 module

### PR DESCRIPTION
**Pull request by MDilipkumar07**
## Issue that this pull request solves

Closes: # (issue number)

### Brief description of what is fixed or changed
Created new table global admin stocks 

## Types of changes

_Put an `x` in the boxes that apply_

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [x] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x] I have performed a self-review of my own code
-   [x] I have created new branch for this pull request
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [ ] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Comment by MDilipkumar07** [comment]
@samay-rgb @PranshuNayak Review

File: FusionIIIT/applications/ps1/models.py [hunk 1]
```diff
@@ -38,4 +38,20 @@ class StockEntry(models.Model):
     bill=models.FileField(blank=False)
 
     class Meta:
-        db_table = 'StockEntry'
\ No newline at end of file
+        db_table = 'StockEntry'
+
+class GlobalStockAdmin(models.Model):
+    s_no = models.IntegerField(null=False)
```
Comment (PranshuNayak) [inline FusionIIIT/applications/ps1/models.py:44]: Elaborate about the field ?

File: FusionIIIT/applications/ps1/models.py [hunk 2]
```diff
@@ -38,4 +38,20 @@ class StockEntry(models.Model):
...
+    item_name = models.CharField(max_length=250, blank=False)
+    department = models.CharField(max_length=100, blank=False)
+    section = models.CharField(max_length=100)
+    floor_no = models.IntegerField(null=False)
+    rooms = models.IntegerField(max_length=100)
+    year = models.IntegerField(null=False)
+    item_id = models.IntegerField(null=False)
+    contd = models.BooleanField
```
Comment (PranshuNayak) [inline FusionIIIT/applications/ps1/models.py:52]: Elaborate about the field ?

**Review by PranshuNayak — COMMENTED** [review]
Remove the scripts folder too.

File: FusionIIIT/applications/globals/migrations/0002_auto_20230127_1451.py
Comment (samay-rgb) [inline FusionIIIT/applications/globals/migrations/0002_auto_20230127_1451.py:1]: Is there any change in models.py of globals application? If not then remove this file.

**Pull request closed without merging**